bmorph/core/workflows.py

```diff
@@ -6,138 +6,36 @@
+def apply_bmorph(raw_ts, train_ts, ref_ts,
+        apply_window, raw_train_window, ref_train_window,
+        condition_ts=None,
+        raw_y=None, train_y=None, ref_y=None,
+        bmorph_interval=pd.DateOffset(years=1),
+        bmorph_overlap=60, n_smooth_long=None, n_smooth_short=5,
+        bw=3, xbins=200, ybins=10,
+        rtol=1e-6, atol=1e-8, method='hist', **kwargs):
+    """Bias correction is performed by bmorph on user-defined intervals.
 
     Parameters
     ----------
     raw_ts : pandas.Series
         Raw flow timeseries.
     train_ts : pandas.Series
         Flow timeseries to train the bias correction model with.
-    obs_ts : pandas.Series
+    ref_ts : pandas.Series
         Observed/reference flow timeseries.
-    train_window : pandas.date_range
-        Date range to train the bias correction model.
-    bmorph_window : pandas.date_range
+    apply_window : pandas.date_range
         Date range to apply bmorph onto flow timeseries.
-    reference_window : pandas.date_range
+    raw_train_window : pandas.date_range
+        Date range to train the bias correction model.
+    ref_train_window : pandas.date_range
         Date range to smooth elements in 'raw_ts' and 'bmorph_ts'.
-    bmorph_overlap : int
-        Total overlap CDF windows have with each other, distributed evenly
-        before and after the application window.
-    n_smooth_long : int, optional
-        Number of elements that will be smoothed in `raw_ts` and `bmorph_ts`.
-        The nsmooth value in this case is typically much larger than the one
-        used for the bmorph function itself. For example, 365 days.
-    n_smooth_short : int, optional
-        Number of elements that will be smoothed when determining CDFs used
-        for the bmorph function itself.
-    train_on_year : boolean, optional
-        Fits a new target CDF for each year in the training period, rather than a
-        single CDF for the entire period. This should only be used for testing
-        purposes.
     raw_y : pandas.Series, optional
         Raw time series of the second time series variable for conditioning.
     train_y : pandas.Series, optional
         Training second time series.
-    obs_y : pandas.Series, optional
+    ref_y : pandas.Series, optional
         Target second time series.
-    bw : int, optional
-        Bandwidth for KernelDensity. This should only be used if `method='kde'`.
-    xbins : int, optional
-        Bins for the first time series. This should only be used if `method='hist'`.
-    ybins : int, optional
-        Bins for the second time series. This should only be used if `method='hist'`.
-    rtol : float, optional
-        The desired relatie tolerance of the result for KernelDensity.
-        This should only be used if `method='kde'`.
-    atol : float, optional
-        The desired absolute tolerance of the result for KernelDensity.
-        This should only be used if `method='kde'`.
-    method : str, optional
```

Comment:
it looks like method got accidentally left out of the next function and therefore cut out completely. please add it to the new `apply_bmorph` docstring